articles/multifactor-authentication/google-auth/admin-guide.md

```diff
@@ -0,0 +1,116 @@
+# Google Authenticator for Administrators
+
+::: panel-info Note:
+This page refers to using Google Authenticator instead of Auth0 Guardian. Google Authenticator can also be used with Guardian(your users choose which to use) when **Push Notifications** is enabled. [Click here for more information on Guardian.](/multifactor-authentication/guardian)
+:::
+
+## Enabling Google Authenticator for MFA
```

Comment:
Shouldn't we warn the users about what would happen if they have guardian or other provider set up?